src/Sylius/Bundle/ProductBundle/Resources/config/doctrine/model/ProductTranslation.orm.xml

```diff
@@ -18,8 +18,8 @@
             <generator strategy="AUTO" />
         </id>
 
-        <field name="name" column="name" type="string" />
-        <field name="slug" column="slug" type="string" unique="true" />
+        <field name="name" column="name" type="string" nullable="true"/>
+        <field name="slug" column="slug" type="string" nullable="true"/>
```

Comment:
You have removed a unique constraint which is effectively an index on slug. We won't have this now so we will need one to be added to replace it and keep searching by slug fast"
```
        <unique-constraints>
            <unique-constraint columns="slug,locale" name="slug_locale_uniq_ind" />
        </unique-constraints>
```